homeassistant/components/isy994/__init__.py

```diff
@@ -181,13 +207,19 @@ async def async_setup_entry(
     def _start_auto_update() -> None:
         """Start isy auto update."""
         _LOGGER.debug("ISY Starting Event Stream and automatic updates")
-        isy.auto_update = True
+        isy.websocket.start()
+
+    def _stop_auto_update(event) -> None:
+        """Stop the isy auto update on Home Assistant Shutdown."""
+        _LOGGER.debug("ISY Stopping Event Stream and automatic updates")
+        isy.websocket.stop()
 
     await hass.async_add_executor_job(_start_auto_update)
 
     undo_listener = entry.add_update_listener(_async_update_listener)
 
     hass_isy_data[UNDO_UPDATE_LISTENER] = undo_listener
+    hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, _stop_auto_update)
```

Comment:
```suggestion
    entry.async_on_unload(hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, _stop_auto_update))
```

Reply:
Should this be a wrap around the stop event listener or should I just use `entry.async_on_unload(_stop_auto_update)`?